api/examples/src/message/claimValue.ts

```diff
@@ -0,0 +1,39 @@
+import { GearApi, GearKeyring, UserMessageRead } from '@gear-js/api';
+
+const main = async () => {
+  const api = await GearApi.create();
+
+  const alice = await GearKeyring.fromSuri('//Alice');
+
+  const mailbox = await api.mailbox.read(GearKeyring.decodeAddress(alice.address));
+
+  const messageWithValue = mailbox.find(([_, message]) => parseInt(message.value) > 0);
+
+  console.log(messageWithValue);
+
+  api.mailbox.claimValue.submit(messageWithValue[0][1]);
```

Comment:
It might be better to create a variable for this `messageWithValue[0][1]`. It will be a little bit easier to understand what it is exactly.

```js
const someVariable = messageWithValue[0][1];
```